lang/es.po

```diff
@@ -9,6 +9,7 @@ msgstr ""
 # Put something like "Latin translation by Jvlivs Gaivs\n" here.
 msgid "TRANSLATION_AUTHOR\n"
 msgstr "Traducción al Español por Mr. Outis\n"
+mgstr "Traducción al Español por Natalie Stroud\n"
```

Comment:
1. `mgstr` -> `msgstr` (the first `s` is missing)
2. Please keep the previous translator's name and add your name in an "X and Y" fashion.